superset/connectors/sqla/views.py

```diff
@@ -167,9 +170,7 @@ class TableModelView(SupersetModelView, DeleteMixin):  # noqa
         'schema': _(
             "Schema, as used only in some databases like Postgres, Redshift "
             "and DB2"),
-        'description': Markup(
-            "Supports <a href='https://daringfireball.net/projects/markdown/'>"
-            "markdown</a>"),
+        'description': _("Supports markdown"),
```

Comment:
Why they cannot be translated? I think you are translating other markdown renderer strings in this very same PR.